package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "mathquill",
+  "main": "build/mathquill.min.js",
```

Comment:
For reference, here is what the directory looks like when installed in the node_modules directory
![image](https://user-images.githubusercontent.com/94476616/165996126-bc628731-0a64-4b94-a634-f490d37312a0.png)

Reply:
Here is the error I get from Vite:
![image](https://user-images.githubusercontent.com/94476616/165996204-1e3af6fe-c3c5-40b2-8609-ed28d0037b90.png)